src/SprykerEco/Zed/Heidelpay/Persistence/Propel/Schema/spy_heidelpay.schema.xml

```diff
@@ -22,6 +22,8 @@
         <column name="date_of_birth" required="false" type="DATE"/>
         <column name="salutation" required="false" type="VARCHAR"/>
 
+        <column name="connector_invoice_account_info" required="false" type="VARCHAR" />
```

Comment:
Are 255 characters enough for that data? Isn't it some sort of JSON?

Reply:
Yes, 255 should be enough. There are only few things like IBAN, bank name and account holder.